contrib/devtools/github-merge.sh

```diff
@@ -82,13 +82,15 @@ function cleanup() {
 }
 
 # Create unsigned merge commit.
+PRTITLE=`curl -s https://api.github.com/repos/$REPO/pulls/$PULL | grep -e '  "title": ".*",'| awk -F'"' '{print $4}'`
```

Comment:
But none of those titles has a " (or quote) symbol in it, or am I missing something?